src/website/app/ComponentDocExample.js

```diff
@@ -83,6 +89,18 @@ export default function ComponentDocExample({
 
   return (
     <Root {...rootProps}>
+      {chromeless &&
```

Comment:
Why does this care about `chromeless`?  Isn't `parent` enough? Or maybe you meant `if (!chromeless)` in order to not add titles and canonical links for `chromeless` urls?